lib/rack/conditional_get.rb

```diff
@@ -56,7 +56,7 @@ def fresh?(env, headers)
     end
 
     def etag_matches?(none_match, headers)
-      etag = headers['ETag'] and etag == none_match
+      headers['ETag'] == none_match
```

Comment:
This changed the behavior. If both `headers['ETag']` and `none_match` are `nil` or false this will return true now, while it returned false with the previous code. If you want to improve this, to keep the same behavior it needs to be.

```ruby
etag = headers['ETag']
etag && etag == none_match
```

Reply:
Thank you for your suggestion.